RxTest/XCTest+Rx.swift

```diff
@@ -170,6 +170,47 @@ public func XCTAssertEqual<T: Equatable>(_ lhs: [Recorded<Event<T?>>], _ rhs: [R
     printSequenceDifferences(lhs, rhs, ==)
 }
 
+/**
+ Assert a list of Recorded events has emitted the provided elements.
+ This method does not take event times into consideration.
+
+ This method will assert a failure if any stop events have been emitted (e.g. `completed` or `error`).
+
+ - parameter stream: Array of recorded events.
+ - parameter elements: Array of expected elements.
+*/
+public func XCTAssertRecordedElements<T: Equatable>(_ stream: [Recorded<Event<T>>], _ elements: [T], file: StaticString = #file, line: UInt = #line) {
+
+    if let stopEvent = stream.first(where: { $0.value.isStopEvent }) {
+        #if os(Linux)
+        XCTFail("A non-next stop event has been emitted: \(stopEvent)")
+        #else
+        XCTFail("A non-next stop event has been emitted: \(stopEvent)", file: file, line: line)
+        #endif
+        return
+    }
+
+    let streamElements = stream.map { event -> T in
```

Comment:
I started with `stream.map { $0.event.value }` but this potentially returns `nil`. 
I thought of doing a `compactMap`, but if the event itself would be `nil` that would cause issues IMO.